src/drivers/driver.ts

```diff
@@ -169,69 +169,56 @@ export abstract class CMakeDriver implements vscode.Disposable {
 
   /**
    * The environment variables required by the current kit
+   * The ${dollar} are not expanded and missing environment variable subs are not expanded for this variables
    */
-  private _kitEnvironmentVariables = new Map<string, string>();
-
-  /**
-   * Compute the environment variables that apply with substitutions by expansionOptions
-   */
-  async computeExpandedEnvironment(in_env: proc.EnvironmentVariables, expanded_env: proc.EnvironmentVariables): Promise<proc.EnvironmentVariables> {
-    const env = {} as {[key: string]: string};
-    const opts = this.expansionOptions;
-
-    await Promise.resolve(
-      util.objectPairs(in_env)
-      .forEach(async ([key, value]) => env[key] = await expand.expandString(value, {...opts, envOverride: expanded_env}))
-    );
-    return env;
-  }
+  private _kitEnvironmentVariables: proc.EnvironmentVariables = {};
```

Comment:
the plan is not to use _ anymore, I know you didn't name this property, but as you are touching it, that would be good to rename it at the end. https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines

Reply:
Sorry, I touched too much variable with leading _, I think it's better to apply the naming guidelines with a single short, maybe by the link rules, otherwise it's would get the code hard to review

Reply:
You are right, it's easier if we start applying new guidelines in future PRs.